README.md

```diff
@@ -108,14 +108,14 @@ Here is a breakdown of the inputs here:
   ```
   (withdrawal_slot - FIRST_CAPELLA_SLOT) // SLOTS_PER_HISTORICAL_ROOT
   ```
-  where `FIRST_CAPELLA_SLOT` on mainnet is 6209536 and `SLOTS_PER_HISTORICAL_ROOT` is 8192. Note that `withdrawal_slot` is the slot number of the block containing the withdrawal you want to prove.
+  where `FIRST_CAPELLA_SLOT` on [Mainnet](https://twitter.com/TimBeiko/status/1755597708520501672) is 6209536, 5193728 on [Goerli](https://twitter.com/TimBeiko/status/1633554636711034880), 950272 on [Hoelsky](https://blog.ethereum.org/2024/01/24/sepolia-holesky-dencun) and `SLOTS_PER_HISTORICAL_ROOT` is 8192. Note that `withdrawal_slot` is the slot number of the block containing the withdrawal you want to prove.
```

Comment:
The first "capella" slot on Holesky is 8192.  You can see [here](https://github.com/eth-clients/holesky?tab=readme-ov-file#metadata) that the Capella epoch is 256.  256*32 = 8192.